src/index.ts

```diff
@@ -501,7 +503,7 @@ async function setFeeOptions({
 	if (gasLimit) {
 		txOptions.gasLimit = gasLimit
 	} else if (txRequest) {
-		const gasLimitRaw = await provider.estimateGas(txRequest)
+		const gasLimitRaw = await provider.estimateGas(peanutToEthersTx(txRequest))
```

Comment:
nice on the 2 converter functions. Thoughts on renaming to peanutToEthersV5Tx?

Reply:
Done!